nextcloud/server
# Show sharees via propfind

:exclamation: TODO: once this is merged, change/check OCFileListAdapter: showShareAvatar boolean :exclamation: 

Steps to test
- create a file "share"
- share this file to userA and userB
- do a propfind with <nc:sharees/>, e.g.:
```
curl --request PROPFIND \
  --url http://localhost/nc/remote.php/webdav/share \
  --header 'authorization: Basic dG9iaTplOGVrei1SbVpaVC1DMkdDTi01clk2My1BeTl5QQ==' \
  --header 'content-type: application/xml' \
  --header 'ocs-apirequest: true' \
  --cookie 'ocjycgrudn78=vgovrq3bgraie6rs6ai2sl42ag; oc_sessionPassphrase=oyqxSU8b6n4N8Aj0yfObGeheppYvsdzgMhEkMgHw76nWBRLVRRM%252FZcjxqQ5QHtm3knScFxIKIjCzDasJYhLEYxRsvabseMjOqDSa%252BNSkI%252Fi%252FDrm6cKlWmA4QBr3NtcGI; nc_sameSiteCookielax=true; nc_sameSiteCookiestrict=true' \
  --data '<?xml version="1.0" encoding="UTF-8"?>
<D:propfind xmlns:D="DAV:">
<D:prop>
	
<sharees xmlns="http://nextcloud.org/ns" />
</D:prop></D:propfind>
'
```
- see: 
	- Federated sharee: `<nc:sharees>tobi@http://localhost/nc15</nc:sharees>`
	- Multiple sharees on same server: `<nc:sharees>admin, Tobi Kaminsky</nc:sharees>`
	- One sharee on same server: `<nc:sharees>admin</nc:sharees>`

Please note that this shows only the user you have created & shared a file to.
The other way around (receiving a share) is already exists via < owner-id> and < owner-display-name>.

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>
